scripts/styleguide-server.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+styleguide \
+  --kss-source 'scss/**/*.scss' \
+  --style-source 'build/bitstyles_styleguide.css' \
+  --output 'styleguide' \
+  --overviewPath 'README.md' \
+  --readOnly \
+  --watch \
+  --server \
+  --disableEncapsulation \
+  --title 'bitstyles' \
+  --customColors "styleguide/assets/stylesheets/styleguide-colors.css" \
+  --extraHead '<script src="/assets/javascripts/ofi.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/a11y-dialog.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/styleguide-init.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/modal.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/expander.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/object-fit-polyfill.js"></script>' \
+  --extraHead '<link rel="stylesheet" type="text/css" href="/assets/stylesheets/styleguide-theme.css">' \
```

Comment:
The `styleguide-server` script seems to duplicate all of the command-line arguments of `styleguide-build` and just adds two more: `--watch` and `--server`.

I think this could be dealt with simply by using `-build` as the base and allow it to receive additional commands. Something like the following should work:

```shell
#!/bin/sh
# styleguide-build

styleguide \
  --kss-source 'scss/**/*.scss' \
  … \
  $*
```

```shell
#!/bin/sh
# styleguide-server

styleguide-build --watch --server
```